server/statistics/avg_over_time.go

```diff
@@ -80,20 +80,25 @@ func (aot *AvgOverTime) Set(avg float64) {
 
 // TimeMedian is AvgOverTime + MedianFilter
 // Size of MedianFilter should be larger than double size of AvgOverTime to denoisy.
-// Delay is aotSize * mfSize * StoreHeartBeatReportInterval /4
+// Delay is aotSize * mfSize * StoreHeartBeatReportInterval/2
+// and the min filled period is aotSize * StoreHeartBeatReportInterval, which is not related with mfSize
 type TimeMedian struct {
 	aotInterval time.Duration
 	aot         *AvgOverTime
 	mf          *MedianFilter
+	aotSize     int
+	mfSize      int
```

Comment:
Where do we use it?

Reply:
`aotSize` will be used in `GetMinFilledPeroid` to get filled period in test

Reply:
I mean `mfSize`

Reply:
I'm not sure whether we will return to the delay later, so I added both of them together